### PR TITLE
Added skip_rows source param

### LINKED ISSUE #75: Support skip rows in goodtables.yml
# Overview

http://goodtables.oklabs.org/jobs/21ff2708-4c66-47af-8406-fcb8ab3a6aa1 (rows 2 and 3 are just commented)

### PR DESCRIPTION
**Pull request by roll**
- fixes #75
- fixed configuration settings parameters naming (camelCase -> snake)

---

- [x] release `tabulator`
- [x] update `tabulator` version here

### CODE REVIEW

**Comment by roll** [comment]
@amercader 
please take a look

File: goodtablesio/schemas/job-conf.yml [hunk 1]
```diff
@@ -90,25 +92,25 @@ properties:
           - schema
 
       # Error limit per table
-      errorLimit:
+      error_limit:
```
Comment (amercader) [inline goodtablesio/schemas/job-conf.yml]: +1 I like this notation much more

File: requirements.txt [hunk 2]
```diff
@@ -46,7 +46,7 @@ requests==2.11.1
 rfc3986==0.4.1            # via jsontableschema
 six==1.10.0               # via datapackage, goodtables, jsonlines, linear-tsv, python-dateutil, tabulator
 sqlalchemy==1.1.4
-tabulator==0.13.0
+tabulator==0.14.0
```
Comment (amercader) [inline requirements.txt]: This should be added to requirements.in, and then run `make requirements` to update this one

Comment (roll) [inline requirements.txt]: Ohh I thought it was goodtables dep

**Comment by roll** [comment]
@amercader 
done

Do `make requirements` work for you?

**Comment by amercader** [comment]
@roll works for me with pip-tools==1.7.0

**Comment by roll** [comment]
Thanks! After pip upgrade it works for me too.

**Pull request merged**